rest_api/index.adoc

```diff
@@ -120,9 +120,9 @@ in the Architecture documentation for a deeper discussion on roles.
 [[rest-api-examples]]
 == Examples
 
-These examples are provided as a reference to provide quick success making 
-REST API calls. They use insecure methods. In these examples a simple `GET` 
-call is made to 
+These examples are provided as a reference to provide quick success making
```

Comment:
These examples provide a quick reference for making successful REST API calls. In these examples,